data/sql/updates/pending_db_world/rev_1616345174221959057.sql

```diff
@@ -0,0 +1,34 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1616345174221959057');
+
+SET
+@POOL            = '11652',
+@POOLSIZE        = '5',
+@POOLDESC        = 'Treasures - Eastern Plaguelands',
+@RESPAWN         = '900',
```

Comment:
minor: no need to use `'` when dealing with numbers

Reply:
You are right, I corrected that. Thanks!